launcher/minecraft/mod/tasks/ModFolderLoadTask.cpp

```diff
@@ -53,33 +53,33 @@ void ModFolderLoadTask::run()
     // Read JAR files that don't have metadata
     m_mods_dir.refresh();
     for (auto entry : m_mods_dir.entryInfoList()) {
-        Mod mod(entry);
+        auto* mod = new Mod(entry);
```

Comment:
LeakBGone : raw pointers = cringe (this is just to deal with some deletes down this function)
```suggestion
        auto mod = Mod::Ptr(new Mod(entry));
```